nginx/nginx.conf.d/coala.io.conf

```diff
@@ -25,7 +25,16 @@ server {
     location /rebase    { return 302 https://coala.io/git#rebasing; }
     location /commit    { return 302 http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html; }
     location /cep       { return 302 https://github.com/coala/cEPs/blob/master/cEP-0000.md; }
+    location /cep1      { return 302 https://github.com/coala/cEPs/blob/master/cEP-0001.md; }
+    location /cep2      { return 302 https://github.com/coala/cEPs/blob/master/cEP-0002.md; }
+    location /cep3      { return 302 https://github.com/coala/cEPs/blob/master/cEP-0003.md; }
+    location /cep5      { return 302 https://github.com/coala/cEPs/blob/master/cEP-0005.md; }
     location /coc       { return 302 https://github.com/coala/cEPs/blob/master/cEP-0006.md; }
+    location /cep9      { return 302 https://github.com/coala/cEPs/blob/master/cEP-0009.md; }
+    location /cep10     { return 302 https://github.com/coala/cEPs/blob/master/cEP-0010.md; }
+    location /cep12     { return 302 https://github.com/coala/cEPs/blob/master/cEP-0012.md; }
+    location /cep13     { return 302 https://github.com/coala/cEPs/blob/master/cEP-0013.md; }
+    location /cep14     { return 302 https://github.com/coala/cEPs/blob/master/cEP-0014.md; }
```

Comment:
can we actually generalize that? Because now we have to register each new cep manually^^

Reply:
actually cep6 is given the link location/coc
wont it be a problem if I generalize it?

Reply:
We can keep the link to cep6 with `coc` additionally to the usual `coala.io/cep6` variant, both are legal imo.

>  wont it be a problem if I generalize it? because link for every other cEP is of the format location/cep#

Exactly because of this it shouldn't be a problem, because we have a standardized format here (`location/cep#`). I'm just not sure if it's easily achievable only using this config file.

Reply:
Ohh
Sorry for asking that silly doubt.
Actually the thought of generalising the links didn't come to my mind. 
 Actually I am new to OS environment. I think a lot before making any commit and just stick to what's described in the issue.

Reply:
If you don't have an idea right now how to do the generalization, it's fine :) we can still open an issue afterwards :+1:

Reply:
So should I close this PR or is it fine if we merge it for now?

Reply:
we can merge ;)

Reply:
Thanks. :)